lineapy/transformer/node_transformer.py

```diff
@@ -188,7 +188,7 @@ def visit_NameConstant(self, node: ast.NameConstant) -> LiteralNode:
             return self.tracer.literal(node.value, self.get_source(node))
 
     def visit_Raise(self, node: ast.Raise) -> None:
-        return super().visit_Raise(node)
+        self._exec_statement(node)
```

Comment:
remove the visitor entirely, our generic visitor should be able to handle all cases of ast.stmt daughter classes (which ast.Raise is)